ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -40,15 +42,22 @@ def hello_endpoint(helloid):
         self.client = Client(self.app, BaseResponse)
 
     def test_simple(self):
-        expected_attrs = {
-            "component": "http",
-            "http.method": "GET",
-            "http.host": "localhost",
-            "http.url": "http://localhost/hello/123",
-            "http.route": "/hello/<int:helloid>",
-            "http.status_code": 200,
-            "http.status_text": "OK",
-        }
+        expected_attrs = BoundedDict.from_map(
```

Comment:
For expectations, I would not use BoundedDict because attributes could get dropped silently.

Reply:
Fixed!